cli/src/build/credentials-manage.ts

```
@@ -0,0 +1,1615 @@
+import { getAppId, getConfig } from '../utils'
+import { onboardingBuilderCommand } from './onboarding/command'
+import { canUseFilePicker, openSaveFilePicker } from './onboarding/file-picker'
+import {
+  clearSavedCredentials,
+  getGlobalCredentialsPath,
+  getLocalCredentialsPath,
+  listAllApps,
+  loadSavedCredentials,
+  removeSavedCredentialKeys,
+  updateSavedCredentials,
+} from './credentials'
```

ESLint

[error] Expected "./credentials" to come before "./onboarding/file-picker".

(perfectionist/sort-imports)